### DISCUSSION TITLE
Reboot ESP32 via remote command

### DISCUSSION THREAD

**Discussion by iridium2001** [jomjol/AI-on-the-edge-device#2220] [Ideas] [answered]
Would it be possible to reboot the ESP32 e.g., via an MQTT command from Home Assistant (or other platforms)? This could be helpful with some less stable devices (e.g. nightly reboot).

**Comment by Slider0007** [comment, marked as answer]
You could use REST API /reboot

Please check documentation: https://jomjol.github.io/AI-on-the-edge-device-docs/REST-API/

**Reply by iridium2001** [reply]
Sorry, I have overlooked that. Thanks for the hint!
